cuentociacua.txt

```diff
@@ -0,0 +1 @@
+Esta es la historia de dos niñas mimadas, prepotentes y fastidiosas que solo les gustaba hablar sobre lo que las otras niñas jugaban. Tanto lo hicieron que perdieron la alegriade vivir.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.